Languages code are now normalized (#11315)
readthedocs/readthedocs.org: docs/user/reference/environment-variables.rst

.. envvar:: READTHEDOCS_LANGUAGE

    The locale name, or the identifier for the locale, for the project being built.
    This value comes from the project's configured language,

>     normalized to be lowercase and use a dash as a separator instead of an underscore.

Reviewer: Don't think we need to change this paragraph, the language should be shown as lowercase everywhere already.

Author: No notes for those who expected to be in the format e.g. pt_BR? Asking this because I faced this issue in Sphinx doc translation and then I landed on this page with this out-of-date info.

>     This value comes from the project's configured language,
>     normalized to be lowercase and use a dash as a separator instead of an underscore.

Reviewer: I thought we mentioned this somewhere else, but doesn't seem like it. So, +1 on having a note about it.

```suggestion
    This value comes from the project's configured language,
    the language code is in lowercase and use a dash as a separator.
```

Author: How about this: 

```suggestion
    This value comes from the project's configured language code,
    which is in lowercase and use a dash as a separator.
```

Alternatively, a "and" before "the language code" to connect the two parts of this sentence.

Reviewer: Sounds good!
